notifications/listener_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 func TestListener(t *testing.T) {
-	registry := storage.NewRegistryWithDriver(inmemory.New())
+	registry := storage.NewRegistryWithDriver(inmemory.New(), distribution.GlobalScope)
```

Comment:
Why does the scope need to be passed in? It doesn't require it. The scope is just a property of it.